react/InputGroup/index.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react'
+import cx from 'classnames'
+import PropTypes from 'prop-types'
+import styles from './styles.styl'
+
+class InputGroup extends Component {
+  constructor(props) {
+    super(props)
+    this.handleFocus = this.handleFocus.bind(this)
+    this.handleBlur = this.handleBlur.bind(this)
+    this.state = {
+      focused: false
+    }
+  }
+
+  handleFocus() {
+    this.setState(() => ({ focused: true }))
+  }
+
+  handleBlur() {
+    this.setState(() => ({ focused: false }))
```

Comment:
```suggestion
    this.setState({ focused: false })
```

It is not necessary to use the functional variant of setState when state is not used to compute the new value of state.